automation/jmeter_ansible_readme.md

```diff
@@ -17,21 +17,21 @@ Use these playbooks to perform actual Jmeter performance test against AMQ Artemi
 No throttling on the producer and consumer side.
 
 ```shell
-ansible-playbook setup_n_run_jmeter_perf_client_playbook.yml --extra-vars "@variable_override.yml"
+ansible-playbook -i hosts/hosts-r1_vpc1.yml playbooks/setup_n_run_jmeter_n_messages_in_parallel_client_playbook.yml --extra-vars "@variable_override.yml"
```

Comment:
@lokeshrangineni This is temporary, right as the test will finally run on the hosts.yml.